game/views/scoreboard.py

```diff
@@ -50,10 +50,12 @@
 from game.models import Level, Attempt, sort_levels
 from portal.models import Class, Teacher, Student
 
-Single_Level_Header = [ugettext('Class'), ugettext('Name'), ugettext('Score'), ugettext('Total Time'),
-                       ugettext('Start Time'), ugettext('Finish Time')]
-Multiple_Levels_Header = [ugettext('Class'), ugettext('Name'), ugettext('Total Score'), ugettext('Total Time'),
-                          ugettext('Progress')]
+Single_Level_Header = [ugettext_lazy('Class'), ugettext_lazy('Name'), ugettext_lazy('Score'),
+                       ugettext_lazy('Total Time'), ugettext_lazy('Start Time'), ugettext_lazy('Finish Time')]
+Progress_Header = ugettext_lazy('Progress')
```

Comment:
Was thinking about something more fancy (e.g. TableHeader class), but since only one header contains a title this seemed to be the simplest solution.